tests/selenium/folder_list.py

```diff
@@ -59,7 +59,7 @@ def show_folders(self):
 
     print("FOLDER LIST TESTS")
     test_runner(FolderListTests, [
-        'reload_folder_list',
+        # 'reload_folder_list',
```

Comment:
Should this be commented again ?

Reply:
It was commented, it's passing by sometimes no. this is unstable i must find a good way that will not be working and no working at an other time.